server.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
```

Comment:
I am for reverting, the name seems odd and if you want to use `npm start` you can overwrite the default script name in the `package.json`